course/static/course/css/parsons.css

```diff
@@ -0,0 +1,9 @@
+.code_lines {
```

Comment:
these two are identical. A better way to do this is to make it one with a more reasonable name like "ts-2".
and if the classes are not used in any other way replace it with ts-2 otherwise add a ts-2 class to the elements you need.